src/configupdater/section.py

```diff
@@ -11,16 +11,15 @@
     the more usual :meth:`~collections.abc.Mapping.get` method of *dict-like* objects.
 """
 import sys
-from typing import TYPE_CHECKING, Optional, TypeVar, Union, cast, overload
+from typing import TYPE_CHECKING, Optional, Tuple, TypeVar, Union, cast, overload
```

Comment:
Related to https://github.com/python/mypy/issues/8715 and https://github.com/python/mypy/issues/10068